cfme/tests/configure/test_zones.py

```diff
@@ -23,25 +23,35 @@ def test_zone_crud(soft_assert):
         casecomponent: WebUI
     """
     zc = current_appliance.collections.zones
+    region = current_appliance.server.zone.region
+
     # CREATE
-    zone = zc.create(
-        name=fauxfactory.gen_alphanumeric(5),
-        description=fauxfactory.gen_alphanumeric(8)
-    )
-    soft_assert(zone.exists, "The zone {} does not exist!".format(
-        zone.description
-    ))
+    name = fauxfactory.gen_alphanumeric(5)
+    description = fauxfactory.gen_alphanumeric(8)
+    zc.create(name=name, description=description)
```

Comment:
Why not do `new_zone = zc.create(name=name, description=description)`? This will avoid the `for` loop below. Or does this create method not return an instance of the object?

Reply:
Added the BZ to the docblock for those two tests. I use the for loop to look up the newly-created zone because the instance returned by the original call to zc.create() does not contain the zone's id or region.

Reply:
@tpapaioa can you pass the zone's id and region to the `create` method?